core/templates/pages/exploration-editor-page/modal-templates/metadata-version-history-modal.component.ts

```diff
@@ -160,14 +160,16 @@ export class MetadataVersionHistoryModalComponent
     if (diffData.oldVersionNumber !== null) {
       this.versionHistoryBackendApiService.fetchMetadataVersionHistoryAsync(
         this.contextService.getExplorationId(), diffData.oldVersionNumber
-      ).then((response: MetadataVersionHistoryResponse) => {
-        this.versionHistoryService.insertMetadataVersionHistoryData(
-          response.lastEditedVersionNumber,
-          response.metadataInPreviousVersion,
-          response.lastEditedCommitterUsername
-        );
-        this.versionHistoryService
-          .incrementCurrentPositionInMetadataVersionHistoryList();
+      ).then((response) => {
+        if (response !== null) {
```

Comment:
If it's null then shouldn't we be handling this in an error callback somehow?

Reply:
This is for the case when the backend api fails for some reason. I was planning to just show an alert to the user in this case and tell them to reload the page once. Will it be okay?

Reply:
Do you mean the orange error messages (rather than the inbuilt browser alert)? If so that is probably fine.

Reply:
Yes. I have added them. PTAL. Thanks!